_spinnaker_user_guides/pipelines.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+order: 15
+redirect_from:
+  - /user-guides/pipelines/
+  - /user_guides/pipelines/
+  - /spinnaker_user_guides/pipelines/
```

Comment:
Should this also include `spinnaker-user-guides/pipelines'?

Reply:
Nope, this creates spinnaker-user-guides/pipelines and doesn't need a redirect to itself.